<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewerHTML</class>
 <widget class="QWidget" name="PreviewerHTML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWebView" name="previewView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="jsCheckBox">
     <property name="toolTip">
      <string>Select to enable JavaScript for HTML previews</string>
     </property>
     <property name="text">
      <string>Enable JavaScript</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ssiCheckBox">
     <property name="toolTip">
      <string>Select to enable support for Server Side Includes</string>
     </property>
     <property name="text">
      <string>Enable Server Side Includes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
